contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics/__init__.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.metrics.export.gauge import Registry
+from opencensus.metrics.export.metric_producer import MetricProducer
+from opencensus.ext.azure.metrics_exporter.standard_metrics.available_memory import AvailableMemoryMetric
```

Comment:
The modules are using different naming rules, it will be great to have some consistency.
1. `available_memory` is adj. + noun.
2. `cpu_usage` is noun. + noun.
3. `process` is noun.

I don't have strong preference here, up to you.